### PR TITLE
Add colorrange attribute to FerriteViz.surface

### PR DESCRIPTION
**Pull request by adarshpalaskar1**
This commit addresses issue #82 by adding the colorrange attribute to FerriteViz.surface

### CODE REVIEW

**Comment by adarshpalaskar1** [comment]
Should I update the docstring as well?

**Comment by koehlerson** [comment]
> Should I update the docstring as well?

Yeah that would be great :)

File: src/makieplotting.jl [hunk 1]
```diff
@@ -312,6 +312,7 @@ values are transformed to a scalar based on `process` which defaults to the magn
 - `scale_plot = false`
 - `shading = false`
 - `colormap = :cividis`
+- `colorrange = (0,1)`
```
Comment (koehlerson) [inline src/makieplotting.jl:315]:
```suggestion
- `colorrange=Makie.automatic`
```

File: src/makieplotting.jl [hunk 2]
```diff
@@ -320,6 +321,7 @@ values are transformed to a scalar based on `process` which defaults to the magn
     scale_plot = false,
     shading = false,
     colormap = :cividis,
+    colorrange = (0,1),
```
Comment (koehlerson) [inline src/makieplotting.jl:324]:
```suggestion
    colorrange=Makie.automatic,
```

**Review by koehlerson — COMMENTED** [review]
Sorry, I just merged the other PR where we added the `Makie.automatic` argument. I think it should work better with this, see https://github.com/Ferrite-FEM/FerriteViz.jl/pull/120

**Comment by adarshpalaskar1** [comment]
> Sorry, I just merged the other PR where we added the `Makie.automatic` argument. I think it should work better with this, see #120

Cool, I have committed the suggestions. I am actually not sure about the conflicting changes.

**Pull request merged**